spec/system/casework/viewing_an_application/application_details/supporting_evidence_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe 'Viewing supporting evidence' do
   include_context 'with stubbed application'
+  include_context 'when downloading a document'
```

Comment:
Why has this changed? is this because there was an unhandled exception?

Reply:
Yes